app.js

```diff
@@ -32,7 +32,7 @@ const activeGames = {};
  */
 initializeDiscordClient();
 const job = new CronJob(
-    '30 0 * * * *',
+    '30 0 0 * * *',
```

Comment:
LG according to the doc: https://github.com/kelektiv/node-cron#cron-ranges
Why wasn't it happening before?

Reply:
I missed that the precision was 'seconds', so basically:
* before: it was running every minute 0, of every hour, on the second 30 (once every hour)
* after: it was running every minute 0, of every hour 0, on the second 30 (once per day)